README.md

```diff
@@ -0,0 +1,2229 @@
+This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
```

Comment:
Maybe eventually we want to remove this README and have our own sexy and small readme.

Reply:
Totally agree, it's the one that create-react-app generates automagically. Lemme kill that bastard